.github/scripts/workflow_to_docker_table.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import argparse
+import pathlib
+
+def parse_command():
+    """Function to parse the command line input
+    Args:
+        None
+    Return:
+        Namespace: returns the args as a standard Namespace object
+    """
+    parser = argparse.ArgumentParser(description='Generate Table of Tools and Their Dockers from a CWL Workflow')
+    parser.add_argument('input_file')
```

Comment:
Does this syntax make it a named positional arg?

Reply:
Correct